DiztinGUIsh/static/Data.cs

```diff
@@ -63,14 +63,27 @@ public const int
         private static ROMMapMode rom_map_mode;
         private static ROMSpeed rom_speed;
         private static List<ROMByte> table;
-        private static Dictionary<int, string> alias, comment;
+        private static Dictionary<int, string> comment;
+
```

Comment:
The new AliasInfo class below is now used for labels.

Before: We stored labels as: Dictionary<int, string> where int is the address, string is the label name
Now: We stored labels as: Dictionary<int, AliasInfo> where int is the address, and AliasInfo holds the label name and the comment

Reply:
A lot of the code changes are little one-offs to deal with pulling this new struct out of the dict.